molecule/delegated/tests/kubectl/redhat.py

```diff
@@ -32,7 +33,10 @@ def test_kubectl_gpg_key_present(host):
     )
 
     k8s_repository_key_url = get_variable(host, "kubectl_redhat_repository_key")
-    key_content = get_from_url(k8s_repository_key_url)
+    k8s_version = get_variable(host, "kubectl_version")
```

Comment:
I would rename `k8s_version` to `kubectl_version`, using two different names here seems very confusing. but then this just duplicates what is done elsewhere in this code, so meh

Reply:
I will merge this as-is and will revise all tests of kubectl with an other PR.